buildkite/src/Jobs/Release/ClientSdk.dhall

```diff
@@ -0,0 +1,38 @@
+let Prelude = ../../External/Prelude.dhall
+
+let S = ../../Lib/SelectFiles.dhall
+let Cmd = ../../Lib/Cmds.dhall
+
+let Pipeline = ../../Pipeline/Dsl.dhall
+let JobSpec = ../../Pipeline/JobSpec.dhall
+
+let Command = ../../Command/Base.dhall
+let OpamInit = ../../Command/OpamInit.dhall
+let Docker = ../../Command/Docker/Type.dhall
+let Size = ../../Command/Size.dhall
+
+in
+
+Pipeline.build
+  Pipeline.Config::{
+    spec = 
+      JobSpec::{
+        dirtyWhen = OpamInit.dirtyWhen # [
+          S.strictlyStart (S.contains "buildkite/src/Jobs/Release/ClientSdk"),
+          S.strictlyStart (S.contains "src")
```

Comment:
can you add another line for `S.strictlyStart (S.contains "frontend/client_sdk")`

Reply:
Done.